samples/highcharts/accessibility/not-accessible-simple-config/demo.html

```diff
@@ -0,0 +1,11 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/export-data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<figure class="highcharts-figure">
+    <div id="container"></div>
+    <p class="highcharts-description">
+        The following chart is demonstrating a very simple configuration, without proper context.
```

Comment:
Is this a demo showing what not to do? If yes, this might not be clearly described.